docs-ref-conceptual/microsoft-graph-migration.md

```diff
@@ -24,22 +24,23 @@ Command argument and behavior breaking changes are listed below.
 
 ### `az ad app create`
 
-- `--reply-urls` argument is split into `--web-redirect-uris` and `--public-client-redirect-uris`
-- `--homepage` argument is replaced by `--web-home-page-url`
-- `--available-to-other-tenants` is replaced by `--sign-in-audience`
-- `--native-app` is replaced by `--is-fallback-public-client`
-- `--oauth2-allow-implicit-flow` is replaced by `--enable-access-token-issuance`
-- `--enable-id-token-issuance` is introduced to set `web/implicitGrantSettings/enableIdTokenIssuance`
-- `--password` is removed. Use `az ad app credential reset` to let Graph create a password for you (https://github.com/Azure/azure-cli/issues/20675)
+- Split `--reply-urls` into `--web-redirect-uris` and `--public-client-redirect-uris`
+- Replace `--homepage` with `--web-home-page-url`
```

Comment:
For the word "replace by/with" combination, active voice favors "with", while passive voice favors "by".

https://english.stackexchange.com/questions/151309/replace-with-versus-replace-by